teku/src/main/java/tech/pegasys/teku/cli/subcommand/TransitionCommand.java

```diff
@@ -180,6 +185,19 @@ private SignedBeaconBlock readBlock(final Spec spec, final String path) throws I
     }
   }
 
+  private void sortBlocksBySlotNumber(final List<SignedBeaconBlock> blocks) {
+    blocks.sort(
```

Comment:
I guess we could do it with 
`blocks.sort(Comparator.comparing(SignedBeaconBlock::getSlot));`

Reply:
My bad!